<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgba(255, 255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>My Inventory management status</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="viewtable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>661</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="DateEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_day">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_day{
	background-color: rgba(204, 204, 204, 255);
	border-radius:5px;
}

QPushButton#btn_day::pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(204, 228, 247, 255);
}</string>
   </property>
   <property name="text">
    <string>Daily</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mon">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_mon{
	background-color: rgba(204, 204, 204, 255);
	border-radius:5px;
}

QPushButton#btn_mon::pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(204, 228, 247, 255);
}</string>
   </property>
   <property name="text">
    <string>Monthly </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quarter">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_quarter{
	background-color: rgba(204, 204, 204, 255);
	border-radius:5px;
}

QPushButton#btn_quarter::pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(204, 228, 247, 255);
}</string>
   </property>
   <property name="text">
    <string>Quarterly</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_year">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_year{
	background-color: rgba(204, 204, 204, 255);
	border-radius:5px;
}

QPushButton#btn_year::pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(204, 228, 247, 255);
}
</string>
   </property>
   <property name="text">
    <string>Yearly</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_main">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_main{
	background-color: rgba(102, 102, 102, 255);
	border-radius:5px;
	color: rgba(255, 255, 255, 255);
}

QPushButton#btn_main::pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(200, 200, 200, 255);
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="past_DateEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>110</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
